### ISSUE TITLE
Empty Trash Plugin

### ISSUE THREAD

**Issue by winkelsdorf** [nate-parrott/Flashlight#391]
Hi,

I don't know where to report an enhancement request for the mentioned plugin:

Please add a shortcut (like "em"). Previously I used to use my own Automator Script, stored in the Application/Utilities Folder which did the same as this plugin. When I typed "em" Spotlight already suggested it as Top Hit, so I just had to press enter to run it.

Using this plugin, I always have to type the whole command "Empty" to get it as Top Hit.

tl;dr
Adding a shortcut command makes it easier to use. Saves letters to type and/or arrows to use.

Cheers

**Comment by nate-parrott** [comment]
Do you think it's worth it to add the ability to define arbitrary shortcuts for any query? (e.g. always rewrite "em" to "empty trash")

**Comment by winkelsdorf** [comment]
Hmm, difficult to answer. I use quite a lot of plugins, in fact Flashlight replaced Alfred for me. That said, some plugins already have good shortcuts, e.g. GutHub Search (gh or github).

Probably it's sufficient to make the plugin developers follow something like a style guide? Writing a whole new shortcut management is imo too much.

There are also some minor inconsistencies when I look at the description of the plugins:
![screen shot 2015-04-05 at 10 46 18](https://cloud.githubusercontent.com/assets/1413291/6996319/082b705a-db81-11e4-8838-cf1494c91303.png)

Note the descriptions. Some developers write two descriptions for the long command and the shortcut, here "stackoverflow xyz"  "so python", some write it like "youtube/yt". Nothing to take care of, but I would suggest them to use the 1st way, as it's easier to read. And suggest they always define a shortcut where it is quite obvious (like gh/github or wiki/wikipedia already did).

In case of Empty Trash I prefer "em" instead of "et" because it's easier to follow the typing flow when using two hands. 

Happy Easter!
Frederik
